packages/@aws-cdk/cx-api/FEATURE_FLAGS.md

```diff
@@ -1376,4 +1379,21 @@ When this feature flag is enabled, specify newly introduced props 's3InputUri' a
 
 **Compatibility with old behavior:** Disable the feature flag to use input and output path fields for s3 URI
 
+
+### @aws-cdk/aws-rds:setCorrectValueForDatabaseInstanceReadReplicaInstanceResourceId
+
+*When enabled, the value of property `instanceResourceId` in construct `DatabaseInstanceReadReplica` will be set to the correct value which is `DbiResourceId` instead of currently `DbInstanceArn`* (fix)
+
+Currently, the value of the property `instanceResourceId` in construct `DatabaseInstanceReadReplica` is not correct, and set to `DbInstanceArn` which is not correct when it is used to create the IAM Policy in the grantConnect method.
+
+When this feature flag is enabled, the value of that property will be as expected set to `DbiResourceId` attribute, and that will fix the grantConnect method.
+
+
+| Since | Default | Recommended |
+| ----- |---------|-------------|
+| (not in v1) |         |             |
+| V2NEXT | `true`  | `true`      |
```

Comment:
The spacing on some of these seem off, was it manually generated or auto generated by running the build?

Reply:
it seems while I merged the conflicts I messed up these files, let me rerun the build and see if it will fix it